CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# High Performance Software Foundation (HPSF) Technical Advisory Software Council (TAC) GitHub Repository Change Process
+
+This document outlines the process for proposing, reviewing, and merging
+changes to the HPSF Technical Advisory Software Council (TAC) repository on
+GitHub. This repository contains critical documents such as the Lifecycle
+Policy for OSS projects and charters for TAC-appointed Working Groups (WGs).
+
+## 1. Proposing Changes
+
+Anyone is welcome and encouraged to propose changes to the HPSF TAC repository.
+The standard method for doing so is by **opening a Pull Request (PR)** on
+GitHub.
+
+## 2. Review and Approval Process
+
+We welcome feedback and suggestions from all members of the community on
+proposed changes. However, formal review and approval for merging are
+restricted to the following:
+
+* **TAC Voting Members:** All current voting members of the TAC are designated
+  with "developer" status on the repository. Their approvals count towards the
+  merging criteria.
+
+Upon opening a PR:
+
+* **Review and Comment:** TAC voting members are expected to review the
+  proposed changes on GitHub, provide comments, and make suggestions as
+  appropriate.
+
+* **Formal Approval:** TAC voting members should use the GitHub "Approve" or
+  "Request Changes" features to formally indicate their assessment of the PR.
+  * "Approve": Indicates that the member believes the changes should be
+    accepted as is.
+  * "Request Changes": Indicates that the member believes the changes require
+    further modification before acceptance.
+
+## 3. Merging Criteria
+
+To ensure velocity in the process while maintaining proper oversight, the
+following merging criteria will generally apply:
+
+* **Minimum Approvals:** A PR generally requires at least two (2) approvals
+  from TAC voting members to be considered for merging.
+
+* **Maintainer Discretion:** If two approvals are met and no TAC member has
+  "requested changes," a designated repository maintainer will exercise
+  judgment on whether to allow more time for additional TAC members to review the
+  changes before merging. This provides flexibility for quick merges when there's
+  clear consensus.
+
+* **Conflict Resolution:** If any TAC voting member "requests changes," these
+  concerns must be addressed and resolved before the PR can be considered for
+  merging.
+
+## 4. Repository Maintainers
+
+The TAC will appoint at least two (2) individuals to serve as repository
+maintainers. These maintainers are responsible for:
+
+* Reviewing PRs that meet the approval criteria.
+* Exercising discretion regarding additional review time.
+* Merging approved PRs into the main branch.
+
+
+## 5. Transparency and Communication
+
+To ensure all TAC members are aware of changes:
+
+* **Bi-Weekly Review:** All changes (merged PRs) that have occurred since the
+  last bi-weekly TAC meeting will be formally brought to the attention of the
```

Comment:
Given that *bi-weekly* has two definitions that are the opposite of one another, [e.g.](https://en.wiktionary.org/wiki/biweekly):

> Adjective
>
> biweekly (not [comparable](https://en.wiktionary.org/wiki/Appendix:Glossary#comparable))
>
>   1. Occurring once every two [weeks](https://en.wiktionary.org/wiki/week#English); [fortnightly](https://en.wiktionary.org/wiki/fortnightly#English).
>   2. Occurring twice a week.

I think we should avoid it and instead use *fortnightly* or *every two weeks*.

Reply:
Also in case someone want to argue that sense 1 is correct, I don't believe correctness is the issue here but rather that both senses are in use, correct or not.

Reply:
```suggestion
* **Bi-Weekly Review:** All changes (merged PRs) that have occurred since the
  last TAC meeting (occurring once every other week) will be formally brought to the attention of the
```